stdlib/reprlib.pyi

```diff
@@ -1,12 +1,14 @@
+import sys
 from array import array
 from collections import deque
 from collections.abc import Callable
-from typing import Any
+from typing import Any, TypeVar
 from typing_extensions import TypeAlias
 
 __all__ = ["Repr", "repr", "recursive_repr"]
 
-_ReprFunc: TypeAlias = Callable[[Any], str]
+_T = TypeVar("_T")
+_ReprFunc: TypeAlias = Callable[[_T], str]
```

Comment:
What's the motivation for this change? It's the only thing making the tests fail :)

Reply:
Ooop.
I thought that `recursive_repr` decorator should not change the type of accepted argument.
Sure, this change is not related to others. I'll revert it.

Reply:
Done